salt/modules/state.py

```diff
@@ -1352,6 +1353,20 @@ def show_state_usage(queue=False, **kwargs):
     return ret
 
 
+def show_states():
+    """Returns the list of states that will be applied on highstate."""
```

Comment:
Could you please break these out into three lines and change the double-quotes to single-quotes? Thanks.

Reply:
This could also use a `.. versionadded:: Oxygen` tag while you're editing this.